src/libraries/System.IO.FileSystem.Watcher/tests/Utility/TestFileSystemWatcher.cs

```diff
@@ -40,5 +45,35 @@ public void CallOnRenamed(RenamedEventArgs e)
         {
             this.OnRenamed(e);
         }
+
+        public void CallNotifyFileSystemEventArgs(WatcherChangeTypes changeType, ReadOnlySpan<char> name)
+        {
+            MethodInfo notifyFileSystemEventArgs = typeof(FileSystemWatcher).GetMethod(
+                "NotifyFileSystemEventArgs",
+                BindingFlags.Instance | BindingFlags.NonPublic,
+                null,
+                new Type[] { typeof(WatcherChangeTypes), typeof(ReadOnlySpan<char>) },
+                null);
+
+            notifyFileSystemEventArgs.CreateDelegate<NotifyFileSystemEventArgs>(this)(changeType, name);
```

Comment:
Would you prefer [`CreateDelegate(Type, Object)`](https://docs.microsoft.com/en-us/dotnet/api/system.reflection.methodinfo.createdelegate?view=net-5.0#System_Reflection_MethodInfo_CreateDelegate_System_Type_System_Object_), so this can be backported?